File: pkg/gate/testdata/ts-proof-pack/fixtures/ts/contract-absence-clean.ts
// contract-absence-clean.ts (TASK-002): a real .ts fixture from which the
// forbidden symbol is genuinely ABSENT, so the TS absence grep probe yields an
// EMPTY result and the gate verdicts PASS (CLM-024). The forbidden token does NOT
// appear anywhere in this file (not even in this comment).
export function cleanTsReplacement(): string {
  return "ok";
}

File: pkg/gate/testdata/contract-absence-clean.go
package contractfix

// contract-absence-clean.go (TASK-001): a real Go fixture from which the
// forbidden symbol is genuinely ABSENT, so the pack's grep absence probe yields
// an EMPTY result and the gate verdicts PASS (CLM-009). The forbidden token does
// NOT appear anywhere in this file (not even in this comment).
func cleanReplacement() string { return "ok" }
